public/app/features/dashboard-scene/saving/SaveProvisionedDashboard.tsx

```diff
@@ -89,9 +103,11 @@ export function SaveProvisionedDashboard({ meta, drawer, changeInfo, dashboard }
           <Input {...register('path')} />
         </Field>
 
-        <Field label="Branch" description="Only supported by GitHub right now">
-          <Input {...register('ref')} />
-        </Field>
+        {isGitHub && (
+          <Field label="Branch" description="Only supported by GitHub right now">
```

Comment:
Do you think that this clarification necessary? I don't think we will support branches for anything else in a very long time.

Reply:
Updated the description.